flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -754,7 +766,9 @@ static void uavoMSPBridgeTask(void *parameters)
 
 	while (1) {
 		uint8_t b = 0;
-		uint16_t count = PIOS_COM_ReceiveBuffer(msp->com, &b, 1, PIOS_QUEUE_TIMEOUT_MAX);
+
+		uint16_t count = PIOS_COM_ReceiveBuffer(msp->com, &b, 1, 3000);
```

Comment:
There's some kind of issue with PIOS_QUEUE_TIMEOUT_MAX (which isn't even a PIOS_COM value) and USB/VCP where it hangs forever even if there's bytes to read.  3000 is chosen to be pretty long / not burden us too much.